lib/identity_cache/should_use_cache.rb

```diff
@@ -0,0 +1,13 @@
+module IdentityCache
+  module ShouldUseCache
+    extend ActiveSupport::Concern
+
+    included do
+      unless self.respond_to?(:should_use_cache?)
+        def self.should_use_cache?
```

Comment:
What do you mean by that? If I have `class ClassMethod` I can't run this. The purpose of this patch is to not override an existing `should_use_cache?`.